test/unit/compile_data/compile_data_test.bzl

```
@@ -65,6 +80,21 @@ def _define_test_targets():
         rustc_flags = ["--cfg=test_compile_data"],
     )
 
+    rust_library(
+        name = "compile_data_env",
+        srcs = ["compile_data_env.rs"],
+        compile_data = ["compile_data.txt"],
+        rustc_env = {
+            "COMPILE_DATA_PATH": "$(location :compile_data.txt)",
```

`location` is soft-deprecated - let's use `execpath` instead:

```suggestion
            "COMPILE_DATA_PATH": "$(execpath :compile_data.txt)",
```